src/features/common/RedeemMicro/RedeemCode.tsx

```diff
@@ -64,23 +64,21 @@ export const InputRedeemCode = ({
           placeholder="XAD-1SA-5F1-A"
           label=""
           variant="outlined"
+          error={errors.code}
+          helperText={errors.code && errors.code.message}
         />
       </div>
 
-      {errors.code && (
-        <span className={styles.formErrors}>{errors.code.message}</span>
-      )}
-
       <div style={{ paddingTop: '30px' }}>
-        <button className={'primaryButton'} onClick={changeRouteCode}>
+        <Button variant="contained" onClick={handleSubmit(changeRouteCode)}>
           {t('redeem:redeemCode')}
-        </button>
+        </Button>
       </div>
     </div>
   );
 };
 
-export const RedeemCodeFailed = ({
+export const RedeemFailed = ({
   errorMessages = null,
```

Comment:
Remove `errorMessages` if it is not going to be used.

Reply:
We decided that in this case we are going to use the message in both places, please take this into consideration.